tests/Examples/openfhe/dot_product_8f_test.cpp

```diff
@@ -0,0 +1,64 @@
+#include <cstdint>
+#include <vector>
+
+#include "gtest/gtest.h"              // from @googletest
+#include "src/pke/include/openfhe.h"  // from @openfhe
+
+// Generated headers (block clang-format from messing up order)
+#include "tests/Examples/openfhe/dot_product_8f_lib.h"
+
+namespace mlir {
+namespace heir {
+namespace openfhe {
+
+CiphertextT dot_product__encrypt__arg0(CryptoContextT v16,
```

Comment:
qq: were the encrypt/decrypt funcs here supposed to be generated by the compiler and in the `dot_product_8f_lib.h` file?

Reply:
Currently we do not have `--add-client-interface` (there is a TODO down below) for CKKS scheme so we need to manually add encrypt/decrypt funcs. The current generated `.h` does not have these signatures.

Reply:
Ah, you're right, thanks! Could you link in this tracking issue so if someone fixes it they can update that code?

Reply:
https://github.com/google/heir/issues/891

Reply:
Reference to #891 has been added in line 49. Should I also add it in this line?

Reply:
Added anyway

Reply:
Anyone is fine! I think they'll find out they can remove it. Thanks!